src/main/resources/reference-data/sscs-venues.csv

```diff
@@ -1,266 +1,266 @@
-1241,"SC293","SSCS Cardiff","Swansea Civil Justice Centre","Caravella House","Quay Parade","Swansea","","SA1 1SP","",301,"https://www.google.com/maps/place/Swansea+Civil+Justice+Centre/@51.6202649,-3.940207,17z/data=!3m1!4b1!4m5!3m4!1s0x486ef54b9d3ec327:0x62e71b5abb876c76!8m2!3d51.6202616!4d-3.9380183","Yes","Swansea Civil Justice Centre",""
-1248,"SC295","SSCS Leeds","Gateshead Law Courts","Warwick Street","","Gateshead","Tyne and Wear","NE8 1DT","",101,"https://goo.gl/maps/kmM4aMvy5hcv8KfY6","Yes","Gateshead Law Courts",""
-1249,"SC100","SSCS Leeds","Hull Combined Court","Lowgate","","Kingston-Upon-Hull","East Yorkshire","HU1 2EZ","",101,"https://goo.gl/maps/vT2jhBJrJ7CmpTEw7","Yes","Hull Combined Court",""
-1253,"SC264","SSCS Cardiff","Worle Queensway","Queensway House","The Hedges","St Georges","","BS22 7BB","",4503,"https://www.google.co.uk/maps/place/Queensway+House,+The+Hedges,+St+Georges,+Weston-super-Mare+BS22+7BB/@51.3608462,-2.9079592,17z/data=!3m1!4b1!4m5!3m4!1s0x4871f9c9b4a6f089:0x8e26bdd0ff4c6e54!8m2!3d51.3608429!4d-2.9057705","No","Worle Queensway","Not active"
-1254,"SC2656","SSCS Bradford","The Law Courts","Easton Street","High Wycombe","Buckinghamshire","","HP11 1LR","",201,"https://www.google.co.uk/maps/place/The+Law+Courts/@51.6276531,-0.7483165,17z/data=!3m1!4b1!4m5!3m4!1s0x487661c568156a73:0x24a76a50cf49f035!8m2!3d51.6276498!4d-0.7461278","Yes","Bristol MC & Tribunal Hrg Centre",""
-1250,"SC297","SSCS Cardiff","Bristol Magistrates Court","Marlborough Street","","Bristol","","BS1 3NU","0117 930 2400",301,"https://goo.gl/maps/Yyq7e2pYdqP2","Yes","Bristol MC & Tribunal Hrg Centre",""
-1256,"SC335","SSCS Bradford","Chelmsford","Chelmsford County & Family Court","Priory Place","New London Road","Chelmsford","CM2 0PP","0344 892 4000",201,"https://www.google.com/maps/place/Chelmsford+County+%26+Family+Court/@51.7307924,0.469755,17z/data=!3m1!4b1!4m5!3m4!1s0x47d8ebd689055555:0x2ba29a3d11684ae5!8m2!3d51.7307924!4d0.4719437","Yes","Chelmsford",""
-1257,"SC336","SSCS Bradford","Hastings Law Courts","Bohemia Rd","","Hastings","","TN34 1ND","",201,"https://goo.gl/maps/NMZ43nC3w8N9jd6B8","Yes","Hastings",""
-1259,"SC228","SSCS Leeds","Newcastle Civil and Family Courts & Tribunal Centre","Barras Bridge","","Newcastle upon Tyne","","NE1 8QF","",101,"https://maps.google.com/maps?q=54.978706,-1.609995","Yes","Newcastle Civic",""
-1261,"SC338","SSCS Glasgow","Kilmarnock (Grand Hall)","Grand Hall","9 Green Street","East Ayrshire","Kilmarnock","KA1 3BN","",501,"https://goo.gl/maps/dV5v3JCXeu6BPBcB6","No","ZZ D U - Kilmarnock (Grand Hall)","Not active"
+000000,"1","SC001","SSCS Leeds","Barnsley SSCS Hearing Venue","Wellington House","","Wellington Street","Barnsley","S70 1WA","0223 391230",101,"https://www.google.com/maps/place/Wellington+House+Wellington+Street+Barnsley+S70+1WA/@53.550589,-1.480475","No","Barnsley","Not active"
```

Comment:
How come you put the ID number as a string and never kept it as an number?

Reply:
It was requested that the ID number should be converted to a string. 
(The most up to date commit also converts the ePIMS ID number to a string)